tests/terraform/runner/resources/plan_root_module_resources_no_values/tfplan.json

```diff
@@ -0,0 +1,201 @@
+              "tags":null,
+              "timeouts":null,
+              "vpc_config":[
+                 {
+                    "endpoint_private_access":false,
+                    "endpoint_public_access":true,
+                    "security_group_ids":null,
+                    "subnet_ids":[
+                       "10.165.77.0/24"
+                    ]
+                 }
+              ]
+           },
+           "after_unknown":{
+              "arn":true,
+              "certificate_authority":true,
+              "created_at":true,
+              "enabled_cluster_log_types":[
+                 false,
+                 false
+              ],
+              "encryption_config":[
+                 
+              ],
+              "endpoint":true,
+              "id":true,
+              "identity":true,
+              "kubernetes_network_config":true,
+              "platform_version":true,
+              "status":true,
+              "version":true,
+              "vpc_config":[
+                 {
+                    "cluster_security_group_id":true,
+                    "public_access_cidrs":true,
+                    "subnet_ids":[
+                       false
+                    ],
+                    "vpc_id":true
+                 }
+              ]
+           }
+        }
+     },
+     {
+        "address":"data.aws_eks_cluster.cluster",
+        "mode":"data",
+        "type":"aws_eks_cluster",
+        "name":"cluster",
+        "provider_name":"registry.terraform.io/hashicorp/aws",
+        "change":{
+           "actions":[
+              "read"
+           ],
+           "before":null,
+           "after":{
+              
+           },
+           "after_unknown":{
+              "arn":true,
+              "certificate_authority":true,
+              "created_at":true,
+              "enabled_cluster_log_types":true,
+              "endpoint":true,
+              "id":true,
+              "identity":true,
+              "kubernetes_network_config":true,
+              "name":true,
+              "platform_version":true,
+              "role_arn":true,
+              "status":true,
+              "tags":true,
+              "version":true,
+              "vpc_config":true
+           }
+        }
+     }
+  ],
+  "configuration":{
+     "root_module":{
+        "resources":[
+           {
+              "address":"aws_eks_cluster.cluster",
+              "mode":"managed",
+              "type":"aws_eks_cluster",
+              "name":"cluster",
+              "provider_config_key":"aws",
+              "expressions":{
+                 "enabled_cluster_log_types":{
+                    "constant_value":[
+                       "audit",
+                       "api"
+                    ]
+                 },
+                 "name":{
+                    "constant_value":"test"
+                 },
+                 "role_arn":{
+                    "constant_value":"arn:aws:iam::204481888888:role/test"
```

Comment:
Can you switch the account I'd with the value 12345678912 ?

Reply:
Just replaced dummy account # in all places.

Reply:
Squashing this PR will leave the old commits as garbage, to be collected in the coming weeks by github 😄